sandbox/handlers/user_details.js

```diff
@@ -0,0 +1,102 @@
+import * as  fs from 'fs'
+import { sendError } from './utils.js'
+
+const paginationOdsCode = 'T00001';
+const odsCodeRegex = new RegExp('^[A-Za-z]\\d{5}$|^[A-Za-z]\\d[A-Za-z]\\d[A-Za-z]$')
+
+export async function user_details(req, res, next) {
+  if (req.headers["authorization"] === "banned") {
+    sendError(
+      res,
+      403,
+      "Request rejected because client service ban is in effect"
+    );
+    next();
+    return;
+  }
+
+  if (req.headers["authorization"] === "server_error") {
+    sendError(
+      res,
+      500,
+      "Something went wrong"
+    );
+    next();
+    return;
+  }
+
+  if (req.headers["authorization"] === "rate_limit") {
```

Comment:
I'm not sure we actually implement 429s in the sandbox application itself, I think the APIM code has cases for, when running in the sandbox, it looks at the "prefer" header and returns that response if included for any endpoint, hence why this isn't implemented for any other endpoint in the sandbox.

Would it be worth checking the behaviour with this code removed? This might also apply to some other error codes like a 500

Reply:
I shall give this a go locally, thank you for the pointer

Reply:
When you push your code to the MR it should automatically deploy to APIM and you can at least pass requests through with the sandbox environment you deploy, do you know how to find the URL?

Reply:
Removing ability to serve 429s and 500s from the sandbox code and instead relying on the APIM route of using the `Prefer` header with `code=500`/`code=429` in a request to get these responses